Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/DocumentObj/SetIshDocumentObj.cs

```diff
@@ -33,100 +34,191 @@ namespace Trisoft.ISHRemote.Cmdlets.DocumentObj
+    ///$ishFolderTopic = Add-IshFolder -ParentFolderId(Get-IshFolder -BaseFolder Data).IshFolderRef -FolderType ISHModule -FolderName "TopicFolder"
+    ///$ishTopicMetadata = Set-IshMetadataField -Name "FTITLE" -Level Logical -Value "My test Topic" |
+    ///                    Set-IshMetadataField -Name "FAUTHOR" -Level Lng -ValueType Value -Value "Username" |
+    ///                    Set-IshMetadataField -Name "FSTATUS" -Level Lng -ValueType Element -Value "VSTATUSDRAFT"
+    ///$ditaTopicFileContent = @"
+    ///&lt;?xml version=&quot;1.0&quot; ?&gt;
+    ///&lt;!DOCTYPE topic PUBLIC &quot;-//OASIS//DTD DITA Topic//EN&quot; &quot;topic.dtd&quot;&gt;
+    ///&lt;topic&gt;&lt;title&gt;Enter the title of your topic here.&lt;/title&gt;&lt;body&gt;&lt;p&gt;This is the start of your topic&lt;/p&gt;&lt;/body&gt;&lt;/topic&gt;
+    ///"@
+    ///$ishObject = Add-IshDocumentObj -FolderId $ishFolderTopic.IshFolderRef -IshType ISHModule -Lng "en" -Metadata $ishTopicMetadata -FileContent $ditaTopicFileContent
+    ///$tempFilePath = (New-TemporaryFile).FullName
+    ///$ditaTopicFileContent.Replace("your topic", "my topic") | Out-File -FilePath $tempFilePath 
+    ///$ishObject = Set-IshDocumentObj -LogicalId $ishObject.IshRef `
+    ///                                -Version $ishObject.version_version_value `
+    ///                                -Lng $ishObject.doclanguage `
+    ///                                -FilePath $tempFilePath `
+    ///                                -Metadata(Set-IshMetadataField -Name "FTITLE" -Level Logical -Value "Updated topic title and content using file path")
+    /// </code>
+    /// <para>Create a topic in a folder and then overwrite its metadata (FTITLE) and blob by providing parameters and FilePath value to the Set-IshDocumentObj cmd-let. New-IshSession will submit into SessionState, so it can be reused by all cmd-lets.</para>
     /// </example>
     [Cmdlet(VerbsCommon.Set, "IshDocumentObj", SupportsShouldProcess = true)]
     [OutputType(typeof(IshDocumentObj))]
     public sealed class SetIshDocumentObj : DocumentObjCmdlet
     {
-
         /// <summary>
         /// <para type="description">The IshSession variable holds the authentication and contract information. This object can be initialized using the New-IshSession cmdlet.</para>
         /// </summary>
-        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = false, ParameterSetName = "ParameterGroup")]
-        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = false, ParameterSetName = "IshObjectsGroup")]
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = false, ParameterSetName = "ParameterGroupFilePath")]
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = false, ParameterSetName = "ParameterGroupFileContent")]
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = false, ParameterSetName = "ParameterGroupIshObjects")]
```

Comment:
The rest of ISHRemote uses `IshObjectsGroup` as parameter set name... this value becomes visible in the editor as a tab in the help (see picture).
![image](https://user-images.githubusercontent.com/19798225/125251746-ef07d700-e2f7-11eb-8d2c-0783c4b88d54.png)

Reply:
Agree, changed back to "IshObjectsGroup"